src/core/streams/actions/morph.js

```diff
@@ -26,7 +26,10 @@ function beforeNodeRemoved(node) {
 }
 
 function beforeNodeMorphed(target, newElement) {
-  if (target instanceof HTMLElement && !target.hasAttribute("data-turbo-permanent")) {
+  if (!(target instanceof HTMLElement)) {
```

Comment:
Could you please change it to wrap with the condition instead of a clause guard:

```js
if (target instanceof HTMLElement) {
...
}
```

I prefer to keep things consistent there, and, also, I think Turbo's codebase tends to avoid guard conditions (that's a just a matter of style preference).

Reply:
Updated in 9c2bd18ff21d49bd9fc3d6df98822689d49c4b49, thanks!